library/pkparse.c

```diff
@@ -124,6 +134,8 @@ int mbedtls_pk_parse_keyfile( mbedtls_pk_context *ctx,
     size_t n;
     unsigned char *buf;
 
+    PK_VALIDATE_RET( ctx != NULL );
```

Comment:
We should check `path != NULL`, too.

Reply:
It was done from `load_file`, but ok, it's clearer that way.